### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by chryzcodez**
Added my GitHub customized profile README.md file

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -520,5 +520,13 @@
     "githubUsername": "webdevrachit",
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
+  },
+  {
+    "name": "Olanrewaju Alaba",
+    "githubUsername": "chryzcodez",
+    "imageUrl": "https://user-images.githubusercontent.com/77321169/106564373-6ce7db00-652d-11eb-9166-f6ee9b63bd29.jpg",
+    "imageurl2": "https://user-images.githubusercontent.com/77321169/106564390-707b6200-652d-11eb-93fe-90e1ec00faa1.jpg",
```
Comment (AllanRegush) [inline data.json:528]: only 1 image please :+1:

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a minor adjustment :nerd_face:

**Comment by chryzcodez** [comment]
> Looking Good 👍 Just a minor adjustment 🤓

Okay, thanks! I already reduced it to a link!
I feel you can make the review again fr the implementation.

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank You :smile:

**Comment by AllanRegush** [comment]
It looks like there is a conflict. Could  you fix this? If you need any help let me know :nerd_face:

**Comment by chryzcodez** [comment]
> It looks like there is a conflict. Could you fix this? If you need any help let me know 🤓

Ok yeah, I updated the profile readme after the pull request I made, so think that's why there was a conflict. I would resolve that now! 
Thanks

**Pull request closed without merging**